webclient/src/md/datenschutz.md

```diff
@@ -1,45 +1,114 @@
 # Datenschutzerklärung
 
-## Verantwortlicher
+## A. Allgemeine Informationen
 
-Verantwortlicher im Sinne der Datenschutzgesetze, insbesondere der EU-Datenschutzgrundverordnung (DSGVO), ist:
-
-Frank Elsinga
+### Name und Kontaktdaten des Verantwortlichen
 
-in Vertretung für das ["TUM-App"-Projektteam](https://www.tum.app/imprint).
+Verantwortlicher im Sinne der Datenschutzgesetze, insbesondere der EU-Datenschutzgrundverordnung (DSGVO), ist:
 
-**Postanschrift:**  
-Josef-Fischhaber-Straße 46  
-82319 Starnberg  
-Deutschland
+Frank Elsing  
+in Vertretung für ["OpenSource @ TUM e.V."](https://tum.dev).
 
-**Kontakt:**  
+Postanschrift: Josef-Fischhaber-Straße 46, 82319 Starnberg
 Telefon: (0049/0)176 – 56–727-579  
-Telefax: (0049/0)89 - 289–18546  
 E-Mail: navigatum (at-symbol) tum.de
 
-## Ihre Betroffenenrechte
+### Kontaktdaten des Datenschutzbeauftragten
 
-Unter den angegebenen Kontaktdaten unseres Datenschutzbeauftragten können Sie jederzeit folgende Rechte ausüben:
+Der/Die Datenschutzbeauftragte der Technischen Universität München
+Postanschrift: Arcisstr. 21, 80333 München
+Telefon: 089/289-17052
+E-Mail: beauftragter(at)datenschutz.tum.de
 
-- Auskunft über Ihre bei uns gespeicherten Daten und deren Verarbeitung (Art. 15 DSGVO),
-- Berichtigung unrichtiger personenbezogener Daten (Art. 16 DSGVO),
-- Löschung Ihrer bei uns gespeicherten Daten (Art. 17 DSGVO),
-- Einschränkung der Datenverarbeitung, sofern wir Ihre Daten aufgrund gesetzlicher Pflichten noch nicht löschen dürfen (Art. 18 DSGVO),
-- Widerspruch gegen die Verarbeitung Ihrer Daten bei uns (Art. 21 DSGVO) und
-- Datenübertragbarkeit, sofern Sie in die Datenverarbeitung eingewilligt haben oder einen Vertrag mit uns abgeschlossen haben (Art. 20 DSGVO).
+### Zwecke und Rechtsgrundlagen für die Verarbeitung
 
-Sofern Sie uns eine Einwilligung erteilt haben, können Sie diese jederzeit mit Wirkung für die Zukunft widerrufen.
-Sie können sich jederzeit mit einer Beschwerde an eine Aufsichtsbehörde wenden, z. B. an die zuständige Aufsichtsbehörde des Bundeslands Ihres Wohnsitzes oder an die für uns als verantwortliche Stelle zuständige Behörde.
-Eine Liste der Aufsichtsbehörden (für den nicht öffentlichen Bereich) mit Anschrift finden Sie unter: https://www.bfdi.bund.de/DE/Service/Anschriften/Laender/Laender-node.html.
+#### Feedback
 
-## Technische Umsetzung
+Die Nutzung des Feedbackformulars erfolgt auf einer rein freiwiligen Basis.
+Für die Nuzung des Feedbackformulars muss den [Tearms of Service und den Datenschutzbestimmungen von GitHub](https://docs.github.com/en/github/site-policy) zugestimmt werden.
+Auf die mögliche Übertragung der Daten außerhalb der Europäischen Union wird hingewisen.
+Für nicht-öffentliche Kommunikation wird auf die Kontaktmöglichkeit über das [Impressum](/about/impressum) hingewiesen.
```

Comment:
Should we mention here also:
```
Die Verarbeitung basiert auf Grundlage des Art. 6 Abs.1 lit. a DSGVO.
```

I'm not a lawyer of course, so I don't know what's required

Reply:
I have added it to be save